src/parser/core/CombatLogParser.test.js

```diff
@@ -240,13 +239,4 @@ describe('Core/CombatLogParser', () => {
       expect(Object.keys(parser._modules).length).toBeGreaterThan(0);
     });
   });
-  describe('emits static events', () => {
```

Comment:
Please maintain this test, it's important that the fightend event being fired is guarded